### PR TITLE
Feature/select g house

### PR DESCRIPTION
**Pull request by philawsophizing**
ref #22

### CODE REVIEW

**Review by Karyum — APPROVED** [review]

File: src/components/search_list.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    props.updateCurrent(guestHouse);
+  };
+  console.log(props);
+
+  return (
+    <div>
+      {props.db.map((guestHouses) => {
+        const boundItemClick = handleClick.bind(this, guestHouses.name);
+        return (
+          <button
```
Comment (jbesraa) [inline src/components/search_list.jsx]: maybe this button should be a different component and then the bind function wont be inside the map function

File: src/containers/select_guesthouse.jsx [hunk 2]
```diff
@@ -3,20 +3,35 @@ import axios from 'axios';
...
+      db: [],
+      currentGuestHouse: {},
     };
+    this.currentGuestHouse = this.currentGuestHouse.bind(this);
+  }
+
+  currentGuestHouse(guestHouse) {
+    const current = this.state.db.filter((curr) => {
+      if (curr.name === guestHouse) {
+        return curr;
```
Comment (jbesraa) [inline src/containers/select_guesthouse.jsx]: curr and current?

**Review by jbesraa — CHANGES_REQUESTED** [review]

**Comment by philawsophizing** [comment]
@esraajb more like this? :maple_leaf:

File: src/containers/select_guesthouse.jsx [hunk 3]
```diff
@@ -3,20 +3,37 @@ import axios from 'axios';
...
+import SearchList from '../components/search_list.jsx';
 
 
 class SelectGuesthouse extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      db: {},
+      title: 'PAS',
+      tagLine: 'Park & Sleep',
```
Comment (jbesraa) [inline src/containers/select_guesthouse.jsx:14]: are title and tagLine ever change? if not, then its pointless to put them in a state.

File: src/containers/select_guesthouse.jsx [hunk 4]
```diff
@@ -3,20 +3,37 @@ import axios from 'axios';
...
+      db: [],
```
Comment (jbesraa) [inline src/containers/select_guesthouse.jsx]: maybe instead of db, use 'guestHousesList'? might be clearer for the reader

**Review by jbesraa — CHANGES_REQUESTED** [review]
looks good 👍, left a couple of comments. Also, no issue referred

File: src/containers/select_guesthouse.jsx (hunk 3 above)
Comment (philawsophizing) [reply src/containers/select_guesthouse.jsx:14]: They are going to be changed (header will update based on these values). Next PR.

File: src/containers/select_guesthouse.jsx (hunk 4 above)
Comment (philawsophizing) [reply src/containers/select_guesthouse.jsx]: Good shout

**Comment by philawsophizing** [comment]
@esraajb ref #22 was in the commit messages :smile: should it be in the comment as well?

Changes made.

**Comment by jbesraa** [comment]
Yea, should be in the description

**Comment by philawsophizing** [comment]
@esraajb Cool cool, done. And will do going forward. Good to merge?

**Pull request merged**